packages/e2e/cypress/integration/ui/components/authenticator/i18n/i18n.steps.ts

```diff
@@ -1,73 +1,7 @@
-// import { translations } from '@aws-amplify/ui';
+import { translations } from '@aws-amplify/ui';
 import { And, Then, When } from '@badeball/cypress-cucumber-preprocessor';
 import { escapeRegExp } from 'lodash';
 
-// temporary workaround due to https://github.com/aws/aws-sdk-js-v3/issues/3828
```

Comment:
Just curious, how can this workaround be removed if https://github.com/aws/aws-sdk-js-v3/issues/3828 is still open?